lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -35,13 +36,14 @@ def github_namespace
       end
 
       def drop
-        @drop ||= MetadataDrop.new(site)
+        @drop ||= MetadataDrop.new
```

Comment:
This makes me feel woozy. I wonder if keeping the `site` on the global GitHubMetadata is a bad idea? I'm ok with `client` and `repository` at that level, but the whole `site` object scares me.